app/controllers/all-service-transactions/download-transactions.js

```diff
@@ -29,6 +29,8 @@ module.exports = (req, res) => {
           time_taken: timestampStreamEnd - timestampStreamStart,
           from_date: filters.fromDate,
           to_date: filters.toDate,
+          gateway_payout_id: filters.gatewayPayoutId,
+          user_id: req.user && req.user.externalId,
```

Comment:
We should really be using the logging keys in pay-js-commons for structured logs. The idea is that we use these for keys we log in multiple places to ensure they all use the same key. For keys that only refer to the single log line, we can just define them here.

Reply:
https://github.com/alphagov/pay-js-commons/blob/master/src/logging/keys.js

Reply:
Great suggestion 👍  didn't realise it had so many keys like https://github.com/alphagov/pay-js-commons/blob/master/src/logging/keys.js#L102. I'll update this